librecad/src/actions/rs_actioneditcopy.h

```diff
@@ -7,7 +7,7 @@
 **
 **
 ** This file may be distributed and/or modified under the terms of the
-** GNU General Public License version 2 as published by the Free Software 
+** GNU General Public License version 2 as published by the Free Software
```

Comment:
Its my configuration on the vim editor. It strips all useless spaces (like at the end of line or before a tab). I have to find out how to revert the file :) Could you simply apply the first part only?